dotCMS/src/main/java/com/dotcms/aspects/interceptors/ContextCloseableMethodInterceptor.java

```diff
@@ -0,0 +1,43 @@
+package com.dotcms.aspects.interceptors;
+
+import com.dotcms.aspects.DelegateMethodInvocation;
+import com.dotcms.aspects.MethodInterceptor;
+import com.dotcms.business.ContextCloseable;
+import com.dotmarketing.db.DbConnectionFactory;
+
+import static com.dotcms.util.AnnotationUtils.getMethodAnnotation;
+
+/**
+ * Method handler for the {@link com.dotcms.business.ContextCloseable} annotation aspect
+ * @author jsanca
+ */
+public class ContextCloseableMethodInterceptor implements MethodInterceptor<Object> {
+
+    public static final ContextCloseableMethodInterceptor INSTANCE = new ContextCloseableMethodInterceptor();
+
+    public ContextCloseableMethodInterceptor() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)